sechub-cli/src/daimler.com/sechub/util/zip.go

```diff
@@ -98,20 +97,30 @@ func zipOneFolderRecursively(zipWriter *zip.Writer, folder string, zContext *zip
 		if err != nil {
 			return err
 		}
-		if zContext.filename == filePath {
+		if zContext.filename == file {
```

Comment:
the variable name `zipContext` would be more consistent as `zipPath` is used in other parts of the code.

Reply:
Agreed. (This naming was from the original author.)
Changed to `zipContext`.